### PR TITLE
chore: fix concurrency execute with melos.

### PR DESCRIPTION
**Pull request by wasabeef**
## What does this change?

I limited concurrent execution because these logs were being output during the execution of the flutter command.

When `melos run get`
```
ERROR: ERROR: [example]: Waiting for another flutter command to release the startup lock...
ERROR: ERROR: [flutter_gen]: Waiting for another flutter command to release the startup lock...
ERROR: ERROR: [flutter_gen_runner]: Waiting for another flutter command to release the startup lock...
```

And fix analyze.

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

Before submitting your PR, there are a few things you can do to make sure it goes smoothly:

- [ ] Make sure to open a GitHub issue as a bug/feature request before writing your code! That way we can discuss the change, evaluate designs, and agree on the general idea
  - [x] Ensure the tests (`melos run test`)
  - [x] Ensure the analyzer and formatter pass (`melos run format` to automatically apply formatting)
- [ ] Appropriate docs were updated (if necessary)

### CODE REVIEW

File: melos.yaml [hunk 1]
```diff
@@ -8,29 +8,31 @@ packages:
 ide:
   intellij:
     enabled: true
-    moduleNamePrefix: ''
-  vscode: true
```
Comment (wasabeef) [inline melos.yaml:12]: `vscode` 
Removed non-existent configuration value.

Comment (AlexV525) [reply melos.yaml:12]: `moduleNamePrefix` should persist, no?

File: melos.yaml [hunk 2]
```diff
@@ -53,23 +55,27 @@ scripts:
...
+      scope:
         - example
         - example_resources
     description: flutter pub run build_runner build --delete-conflicting-outputs
 
   test:
-    run: melos test:dart --no-select && melos test:flutter --no-select
+    run: |
+      melos test:dart --no-select && \
+      melos test:flutter --no-select
```
Comment (AlexV525) [inline melos.yaml:78]: We don't need `&& \` if the command is not inlined, probably

File: melos.yaml (hunk 1 above)
Comment (wasabeef) [reply melos.yaml:12]: OK, I don't have a strong preference regarding this, so I'll leave it as it is.

Comment (AlexV525) [reply melos.yaml:12]: This helps when using the IDEA so those module won't have a redundant `melos` prefix.

**Review by AlexV525 — APPROVED** [review]
👍🏻

**Pull request merged**